cla-backend-go/v2/cla_manager/service.go

```diff
@@ -556,6 +557,42 @@ func (s *service) CreateCLAManagerDesignee(companyID string, projectID string, u
 	return claManagerDesignee, nil
 }
 
+//CreateCLAManagerDesigneeByGroup creates designee by group for cla manager prospect
+func (s *service) CreateCLAManagerDesigneeByGroup(params cla_manager.CreateCLAManagerDesigneeByGroupParams, projectCLAGroups []*projects_cla_groups.ProjectClaGroup, f logrus.Fields) ([]*models.ClaManagerDesignee, string, error) {
+	var designeeScopes []*models.ClaManagerDesignee
+	foundationSFID := projectCLAGroups[0].FoundationSFID
+	if foundationSFID != "" {
+		claManagerDesignee, err := s.CreateCLAManagerDesignee(params.CompanySFID, foundationSFID, params.Body.UserEmail.String())
+		if err != nil {
+			if err == ErrRoleScopeConflict {
```

Comment:
Could you change this check to err == ErrCLAManagerDesigneeConflict as thats the error conflict that the designee service returns